docs/configuration.md

```diff
@@ -41,6 +41,10 @@ schema:
 # Configures whether we obfuscate graphql-protect validation errors such as max_aliases/max_tokens
 # Recommended to set it to 'true' for public environments
 obfuscate_validation_errors: false    
```

Comment:
maybe make a config for 
`response_errors`
and add options for disabling `validation_errors` and `upstream_errors`?

Reply:
i like that idea

Reply:
ended up solving a bit different since these 2 component are used in totally different places in the codebase it (atleast for now) made more sense to do it like this